quiche/src/recovery/bbr/mod.rs

```diff
@@ -303,10 +303,10 @@ fn on_packets_acked(
         packets.drain(..).fold(0, |acked_bytes, p| {
             r.bbr_state.prior_bytes_in_flight = r.bytes_in_flight;
 
-            per_ack::bbr_update_model_and_state(r, &p, now);
-
             r.bytes_in_flight = r.bytes_in_flight.saturating_sub(p.size);
 
+            per_ack::bbr_update_model_and_state(r, &p, now);
+
```

Comment:
This might change the result of functions using bytes_in_flight in bbr_update_model_and_state(). Is this a necessary change?